web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-typography.php

```diff
@@ -0,0 +1,466 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ct7mm7.php, but found phpcs-scan-cT7MM7.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).